feeder/feeder.go

```diff
@@ -33,7 +35,24 @@ var configFile = "/etc/container-feeder.json"
 
 // special type to load the container-feeder.json config
 type FeederConfig struct {
-	Target string `json:"feeder-target,omitempty"`
+	Target    string   `json:"feeder-target,omitempty"`
+	Whitelist []string `json:"whitelist,omitempty"`
+}
+
+// parseWhitelist returns a whitelist with normalized elements.
+func parseWhitelist(whitelist []string) ([]string, error) {
+	list := []string{}
+	for _, w := range whitelist {
+		name, tag, err := normalizeNameTag(w)
+		if err != nil {
+			return nil, fmt.Errorf("error parsing whitelist item '%s': %v", w, err)
+		}
+		if tag != "" {
```

Comment:
Are  you sure this is not going to add the `:latest` tag to all the whitelist filters?

Reply:
Yes, I covered that in the unit test line 45.